oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/MQEndpointAccess.java

```diff
@@ -47,10 +52,6 @@ public String getEntityId() {
     @ScopeDefaultColumn.DefinedByField(columnName = "endpoint", requireDynamicActive = true)
     private String endpoint;
 
-    @Getter
-    @ScopeDefaultColumn.DefinedByField(columnName = "service_id")
-    private String serviceId;
```

Comment:
I remember we need this field in final entity to support endpoint top N query with parent service ID, don't we?

Reply:
Sorry , what's  `final entity  ` ? , I couldn't  understand it   ,  I don't provide top N metrics  at least now .  and  I remove `serviceId` because it  dose not affect virtual mq metrics

Reply:
You don't provide, but you should make sure it is supported, as it should from protocol perspective.
All endpoint and instance metrics should be.

Please recheck all recent added sources.

Reply:
Yes,  I have checked other virtual service , only mq have endpoint/instance